wallets-testing/test/test.service.ts

```diff
@@ -1,19 +1,29 @@
 import { configService } from '../config';
-import { EthereumNodeService } from '@lidofinance/wallets-testing-nodes';
 import { ETHEREUM_WIDGET_CONFIG } from '@lidofinance/wallets-testing-widgets';
-import { ExtensionService } from '@lidofinance/wallets-testing-extensions';
-import { BrowserContextService, BrowserService } from '../browser';
+import { BrowserService } from '@lidofinance/browser-service';
+import { METAMASK_COMMON_CONFIG } from '@lidofinance/wallets-testing-wallets';
 
 export async function initBrowserService() {
-  const ethereumNodeService = new EthereumNodeService({
-    chainId: ETHEREUM_WIDGET_CONFIG.chainId,
-    rpcUrl: configService.get('RPC_URL'),
-    defaultBalance: 1000,
+  METAMASK_COMMON_CONFIG.LATEST_STABLE_DOWNLOAD_LINK = undefined;
```

Comment:
why do we have to init it here?

Reply:
because we dont need latest stable version for wallet testing